packages/unchained-client/src/types.ts

```diff
@@ -1,6 +1,8 @@
+// these are user facing values, and should be rendered as such
```

Comment:
@kaladinlight plz confirm this comment is valid, and that we don't rely on these internally for anything, and they're purely presentational.

they're referenced in web here https://github.com/shapeshift/web/pull/2232/files#diff-2575b4bbecd755e30395862fd46ebb14c318ded832e13b7b14edd17cb4251c54L57

tests pass but 🤷

Reply:
That is correct, they are added as ways to narrow the trade type to a specific dex if we had different UI or it looks like we are using those strings verbatim for UI display. Nothing internally